warrior_bot/utils/faculty_lookup.py

```diff
@@ -22,13 +22,16 @@
 from dataclasses import dataclass
 from typing import Generator
 from urllib.request import urlopen
+from difflib import get_close_matches
 
 from bs4 import BeautifulSoup
 
 
 class StaffExtractor:
     DIR_URL = "https://wayne.edu/people?type=people&q="  # Directory search URL
     STAFF_URL = "https://wayne.edu/people/"  # Base URL for staff profiles
+    MAX_PAGES = 5 # Amount of pages _fetch_soup_dir will look through.
+    MAX_SEARCH_OPTIONS = 50 # Given options to the user
```

Comment:
```suggestion
    MAX_SEARCH_OPTIONS = 50 # Given options to the user
    CACHE_FILE = "faculty_cache.json"
    BULLETIN_URL = "https://bulletins.wayne.edu/faculty/"
```

These should go here since we're actually now going to parse that link to extract staff instead of searching